iceoryx_examples/waitset/ice_waitset_gateway.cpp

```diff
@@ -24,20 +24,24 @@
 #include <chrono>
 #include <iostream>
 
+constexpr uint64_t NUMBER_OF_SUBSCRIBERS = 2U;
+
 std::atomic_bool keepRunning{true};
```

Comment:
Is there a specific reason why you keep the `atomic bool` here and the following waitset examples, but changed it everywhere else to a `volatile bool`?